### PR TITLE
GNZ-1351: Update README to better illustrate what genezio does

### PR DESCRIPTION
**Pull request by vladiulianbogdan**
# Pull Request Template

## Type of change

-   [x] 📝 Documentation Update

## Description

Update README to better illustrate what is genezio

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [x] I have updated the documentation;
-   [x] I have added tests;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,20 +64,26 @@ For more details on how to use `genezio`, you can check out the official [docume
...
--   🪛&nbsp; Deploy your backend in no-time on a serverless infrastructure.
--   🖼️&nbsp; Host your frontend on the genezio infrastructure.
--   🪄&nbsp; A magically-generated SDK to call your server functions from the client (supported languages: JS, TS, Swift, Python, Dart).
--   👀&nbsp; A handful of examples to start from at [genezio-examples](https://github.com/Genez-io/genezio-examples/).
--   🔨&nbsp; A dedicated GitHub Action to integrate in your CI/CD at [genezio-github-action](https://github.com/Genez-io/genezio-github-action).
+-   💡 Full static type-safety with **auto-completion** in your favorite editor.
+-   🧩 **Typesafe RPC** implemented by parsing the backend code, storing the AST - to be able to **generate the SDK in any frontend language with zero dependencies**.
+-   📦 Export the resulting SDK using dependency managers such as **NPM Registry** - private to your team or public for anybody to use.
+-   🌐 **Framework agnostic**: works with React, Vue, Angular, Svelte, ExpressJS, Fastify, etc.
+-   🎯 **DX enhanced bundling and compiling**: no more .
```
Comment (costinsin) [inline README.md:79]: I don't quite understand this statement

File: README.md [hunk 2]
```diff
@@ -71,20 +64,26 @@ For more details on how to use `genezio`, you can check out the official [docume
...
 # Features
 
--   👩‍💻&nbsp; Write your backend code in Javascript, Typescript or Dart.
--   🪛&nbsp; Deploy your backend in no-time on a serverless infrastructure.
--   🖼️&nbsp; Host your frontend on the genezio infrastructure.
--   🪄&nbsp; A magically-generated SDK to call your server functions from the client (supported languages: JS, TS, Swift, Python, Dart).
--   👀&nbsp; A handful of examples to start from at [genezio-examples](https://github.com/Genez-io/genezio-examples/).
--   🔨&nbsp; A dedicated GitHub Action to integrate in your CI/CD at [genezio-github-action](https://github.com/Genez-io/genezio-github-action).
+-   💡 Full static type-safety with **auto-completion** in your favorite editor.
+-   🧩 **Typesafe RPC** implemented by parsing the backend code, storing the AST - to be able to **generate the SDK in any frontend language with zero dependencies**.
```
Comment (costinsin) [inline README.md:76]: > storing the AST - to be able to **generate the SDK
This sounds a bit odd to me

File: README.md [hunk 3]
```diff
@@ -155,7 +154,6 @@ The genezio CLI tool supports the commands shown in the following table:
 
 | Command                                        | Description                                                                                                                                                                                                  |
 | ---------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (costinsin) [inline README.md:156]: Add `genezio` and `genezio create` commands. Copy the content from this page: https://docs.genez.io/genezio-dev-documentation/~/changes/0mNx5jxxocT3LaouL9eE/cli-tool/cli-commands

File: README.md [hunk 4]
```diff
@@ -43,14 +43,7 @@
...
 
-[genezio](https://genez.io/) is a platform for developers to write and host a serverless application.
-
-You are writing your backend logic in classes and design your frontend using your preferred programming language. More languages coming soon! Better yet, join our open source community and help us build it!
-
-By calling `genezio deploy`, we are deploying your backend classes in a serverless infrastructure.
-
-To make it easy to call your backend logic from the client, an SDK is generated.
-You can use it to remotely call the methods of your class in a very natural way.
+[genezio](https://genez.io/) is a developer platform for full-stack developers or teams who need to build, run and maintain web, mobile or enterprise apps with a typesafe backend that scales automatically.
```
Comment (bogdanripa) [inline README.md:46]: Maybe something like - full stack developers that care about type safety?

File: README.md [hunk 5]
```diff
@@ -79,12 +72,21 @@ For more details on how to use `genezio`, you can check out the official [docume
...
 # Features
 
--   👩‍💻&nbsp; Write your backend code in Javascript, Typescript or Dart.
--   🪛&nbsp; Deploy your backend in no-time on a serverless infrastructure.
--   🖼️&nbsp; Host your frontend on the genezio infrastructure.
--   🪄&nbsp; A magically-generated SDK to call your server functions from the client (supported languages: JS, TS, Swift, Python, Dart).
--   👀&nbsp; A handful of examples to start from at [genezio-examples](https://github.com/Genez-io/genezio-examples/).
--   🔨&nbsp; A dedicated GitHub Action to integrate in your CI/CD at [genezio-github-action](https://github.com/Genez-io/genezio-github-action).
+-   💡 Full static type-safety with **auto-completion** in your favorite editor.
+-   🧩 **Typesafe RPC**: Implemented through parsing the backend code and storing the AST, this allows for generating the SDK in any frontend language
```
Comment (bogdanripa) [inline README.md:76]: AST - this is a technicality. Focus on the fact that we do type safety and auto-complete in your IDE on top of multiple backend programming languages like TypeScript, Go, etc

**Pull request merged**